src/style.css

```diff
@@ -0,0 +1,40 @@
+* {
```

Comment:
will be better to add do not need to add it to all elements, body will be enough